ext/bin/pdb-dataset

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+usage() { echo 'Usage: PDBBOX=BOX_DIR pdb-dataset import --timeshift-to now DUMPFILE'; }
+
+if test "$#" -lt 1 -o -z "$PDBBOX"; then
+    usage 1>&2
+    exit 1
+fi
+
+misuse()
+{
+    usage 1>&2
+    exit 1
+}
+
+timeshift=''
+timeshiftsql=''
+dumpfile=''
+command=''
+while test $# -gt 0; do
+    case "$1" in
+        --timeshift-to)
+            test $# -gt 0 || misuse
+            timeshift="$2"
+            shift 2
+            ;;
+        *)
+            if [ -z $command ]; then
+              command="$1"
+              shift
+            elif [ -z $dumpfile ]; then
+              dumpfile="$1"
+              shift
+            else
+              echo "Unknown argument:" $1
+              misuse
+            fi
+    esac
+done
+
+case $command in
```

Comment:
Also, it looks like this case statement is indented 2 spaces, but the previous one is indented 4.  I'm not sure what we want to do more broadly, but might be worth being consistent within the same file, so I'd suggest adjusting one or the other, so they match.